ansible/cloud_providers/ec2_infrastructure_deployment.yml

```diff
@@ -62,6 +62,17 @@
       import_role:
         name: infra-common-ssh-config-generate
 
+    - name: Stop SSH/Socks proxy for Windows proxying through bastion
```

Comment:
Should these tasks be in the bastion role instead?

Reply:
We do a wait on windows hosts before the bastion role is executed IIRC, so we need this proxy in place early.